pom.xml

```diff
@@ -489,6 +495,19 @@
                         </excludes>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <executions>
+                        <execution>
```

Comment:
What's the purpose of configuring `default-jar` here?

Reply:
It isn't really required as I don't think JDBI builds any shaded jars. The reason it is there is I lifted it from another POM I have which sets other settings for the plugin as well and this was required to get everything working for that particular build. I removed most of the custom settings and neglected to remove this one. I will remove it.